aptoide-installer/src/main/java/cm/aptoide/pt/installer/AptoideDownloader.kt

```diff
@@ -32,7 +32,6 @@ class AptoideDownloader @Inject constructor(
         if(installPackageInfo.hasObb()) {
           installPermissions.checkIfCanWriteExternal()
         }
-        installPermissions.checkIfCanInstall()
```

Comment:
I don't think we should do this, as this could break the installations

Reply:
@G0retZ we check on the installations part, as it's done on vanilla. After the app is downloaded, if the user doesn't have install unknown sources permission, our dialog for the install unknown sources will appear

Reply:
Should i create a method to checkIfCanInstall but without asking for the permissions?